src/gdcdictionary/schemas/follow_up.yaml

```diff
@@ -1595,8 +1595,11 @@ properties:
     $ref:
```

Comment:
Verified that this is synced with evidence_of_recurrence_type

Reply:
double verified.